workitem/typegroup/typegroup.go

```diff
@@ -54,3 +55,14 @@ var Requirements0 = WorkItemTypeGroup{
 		workitem.SystemBug,
 	},
 }
+
+// Execution0 defines first level of typegroup Execution
+// This group has less priority than Requirements
+var Execution0 = WorkItemTypeGroup{
+	Group: GroupExecution,
+	Level: []int{2, 0},
```

Comment:
Hi @hectorj2f yeah, True.
I have added description https://github.com/fabric8-services/fabric8-wit/pull/1569/commits/5b82e514a8f7c54703b966ef0759cf6cb88730aa
Let me know if it is understandable.